extensions/git/package.json

```diff
@@ -1893,6 +1893,25 @@
           "default": [],
           "scope": "resource"
         },
+        "git.branchProtectionIndicator": {
+          "type": "object",
```

Comment:
I'm not too sure using `object` for stuff like this is recommended. Might be best to simply have two separate settings. Makes events and comparisons easier. Ie `git.branchProtectionIndicator.branchPicker` and `git.branchProtectionIndicator.statusBar`.

Reply:
Object seems to be more convenient instead of two boolean settings due to the rendering in the Settings editor:

<img width="935" alt="image" src="https://user-images.githubusercontent.com/3372902/174390022-e2de3a35-531b-406c-9151-2dae364a435d.png">